### DISCUSSION TITLE
Shelly dimmer 2 no full brightness

### DISCUSSION THREAD

**Discussion by jeroenst** [arendst/Tasmota#12617] [Q&A] [answered]
I installed a fresh shelly dimmer 2 with tasmota but when I set brightness to 100% the lamp is about 50% on.

Dimmerrange is set from 0 to 100.

What can cause this problem?

**Comment by ascillato2** [comment]
Have you tried the proposed solutions to this on https://github.com/arendst/Tasmota/issues/10202 and https://github.com/arendst/Tasmota/issues/9412 ?

**Reply by jeroenst** [reply]
I can't find any information about the issue I experience. I will try another bulb today, maybe that's the issue...

**Comment by jeroenst** [comment, marked as answer]
Dimmerrange is not 0..100 but 0..255.

Dimmerrange 70,200 is the sweet spot for me.

Documentation should be adapted imo

**Reply by Jason2866** [reply]
Doc changed, thx for pointing to this

**Reply by jeroenst** [reply]
Thanks!

**Reply by jeroenst** [reply]
Maybe the maximum is even 1000 instead of 255 according to https://github.com/jamesturton/shelly-dimmer-stm32/blob/main/COMMUNICATION.md#switch-command---0x01
